### PR TITLE
Documentation Class Instantiation

### PR DESCRIPTION
**Pull request by FORTE-WP**
Related to #1722

### CODE REVIEW

File: WordPress/Docs/Classes/ClassInstantiationStandard.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        </code>
+        <code title="Invalid: without parenthesis.">
+        <![CDATA[
+$a = new Foobar;
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    Don't use spaces between object name and open parenthesis when instantiating new object.
```
Comment (jrfnl) [inline WordPress/Docs/Classes/ClassInstantiationStandard.xml]:
```suggestion
    Don't use spaces between the object name and the open parenthesis when instantiating new object.  
```

File: WordPress/Docs/Classes/ClassInstantiationStandard.xml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    Don't use spaces between object name and open parenthesis when instantiating new object.  
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: no spaces between object name and parenthesis.">
```
Comment (jrfnl) [inline WordPress/Docs/Classes/ClassInstantiationStandard.xml]:
```suggestion
        <code title="Valid: no whitespace between the object name and the parenthesis.">
```

File: WordPress/Docs/Classes/ClassInstantiationStandard.xml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        <![CDATA[
+$a = new Foobar();
+        ]]>
+        </code>
+        <code title="Invalid: spaces between object name and parenthesis.">
```
Comment (jrfnl) [inline WordPress/Docs/Classes/ClassInstantiationStandard.xml]:
```suggestion
        <code title="Invalid: a space between the object name and the parenthesis.">
```

File: WordPress/Docs/Classes/ClassInstantiationStandard.xml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        <![CDATA[
+$a = new Foobar<em> </em>();
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    Object instantiation by reference is not supported by PHP anymore.
```
Comment (jrfnl) [inline WordPress/Docs/Classes/ClassInstantiationStandard.xml]: I know I wasn't too clear about what this error is about on Friday, so I've looked up a better/more accurate description now.

```suggestion
    Assigning the return value of "new" by reference was deprecated in PHP 5.3 and removed in PHP 7.0. New by reference should no longer be used.
```

File: WordPress/Docs/Classes/ClassInstantiationStandard.xml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    Object instantiation by reference is not supported by PHP anymore.  
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: object instantiation without reference.">
+        <![CDATA[
+$a = new Foobar();
```
Comment (jrfnl) [inline WordPress/Docs/Classes/ClassInstantiationStandard.xml]:
```suggestion
$a = <em>new</em> Foobar();
```

File: WordPress/Docs/Classes/ClassInstantiationStandard.xml [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+        ]]>
+        </code>
+        <code title="Invalid: object instantiation by reference.">
+        <![CDATA[
+$a = <em>&</em> new Foobar();
```
Comment (jrfnl) [inline WordPress/Docs/Classes/ClassInstantiationStandard.xml]:
```suggestion
$a = <em>& new</em> Foobar();
```

**Comment by FORTE-WP** [comment]
I just processed all feedback from jrfnl in a new commit.

**Comment by jrfnl** [comment]
Thank you @FORTE-WP for making those changes!

**Pull request merged**